### PR TITLE
Create build-the-builder.md

### LINKED ISSUE #2990: Document the make targets and environment variables required to build the CDI builder container
**Description**:
Document how to build the builder helper with bazel on various architectures

**What you expected**:
There are undocumented make targets and environment variables, and the build scripts invoke environment variables that are undocumented, and set defaults which are inappropriate for building test builder containers and for architectures other than x86.

**URL**:
Provide a link to the documentation: there is none for this aspect of the KubeVirt CDI tool. That's the problem. 

**Additional context**:
This procedure will be tested on an s390x Ubunu LPAR, with the test builder container to be stored in IBM cloud container storage so that _all_ the environment variable settings can be tested, as these are at variance with the default settings of an x86 build container object format and quay.io registry.

### PR DESCRIPTION
**Pull request by cfilleke**
Document the manual build of CDI builder helper container and typical make targets

**What this PR does / why we need it**:

provide documentation for the manual build of the CDI builder helper container and various environment variables that affect the manual build and make target 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2990 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by awels — COMMENTED** [review]
Note you only need the builder if you are not cross compiling to your target environment. For instance for arm64 we cross compile the binary for arm64 using an amd64 builder image. We just tell it to cross compile.

**Comment by kpol-lgx** [comment]
Thanks for your efforts! I have tried to build arm64 version on arm64v8 CPU and it successes.
But I find that the value of  `BUILD_ARCH` should be `aarch64` rather than `arm64` in my machine. So I suggest that add `aarch64` to options list. 

In summary, switching between `arm64` and `aarch64` solves potential problems

File: doc/build-the-builder.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Build The KubeVirt CDI Builder Container
+
+## Native Build toolchain
+
+1. Install the [prerequisites](https://github.com/kubevirt/kubevirt/blob/main/docs/build-the-builder.md#prerequisites) as in the instructions for building the KubeVirt builder container.
+2. Also install `jq` if the docker build utility you use is `podman-buildah`.  
+3. Build the KubeVirt builder container if you haven't already
```
Comment (awels) [inline doc/build-the-builder.md:7]:
```suggestion
3. Build the containerized data importer builder container if you haven't already
```

**Pull request merged**